brambling/utils/model_tables.py

```diff
@@ -374,11 +360,19 @@ def get_custom_fields(self):
     def _get_custom_fields(self):
         raise NotImplementedError
 
+    def _show_housing_data(self, attendee, form_entry):
```

Comment:
This doesn't seem like it should be on CustomDataTable, since it only actually has an effect on the AttendeeTable. In fact it's a little misleading because the second arg will only be an attendee if this *is* an attendee table.

Maybe a _get_custom_data method instead, which returns what becomes raw_data? Then AttendeeTable could override to add this filter, and OrderTable could override to add the data from eventhousing.